02/app.js

```diff
@@ -0,0 +1,40 @@
+const formEl = document.querySelector('form');
+formEl.noValidate = true;
+
+formEl.addEventListener('submit', checkData);
+
+function checkData(e) {
+	e.preventDefault();
+	const email = e.target.elements.login;
+	const pass1 = e.target.elements.pass1;
+	const pass2 = e.target.elements.pass2;
+	const accept = e.target.elements.accept;
+	const errors = [];
+
+	email.previousElementSibling.style.color = 'black';
+	pass1.previousElementSibling.style.color = 'black';
+	pass2.previousElementSibling.style.color = 'black';
+	accept.previousElementSibling.style.color = 'black';
```

Comment:
Zwróć uwagę, że mamy tutaj powielenie kodu.
Może lepiej użyć pętli? -> https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements

PS. Raczej bym pogrupował podobnie działania - jak nie funkcja to przynajmniej przerzuciłbym ten element na dół (przed ifem). Wtedy mam "resetowanie" koloru, a potem odpowiednie kolorowanie na czerwono.